server/strategy.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // DecideOps returns the next operations to do and the operation phase.
 // This returns nil when no operations need to be done.
-func DecideOps(c *cke.Cluster, cs *cke.ClusterStatus, constraints *cke.Constraints, resources []cke.ResourceDefinition, reboot *cke.RebootQueueEntry) ([]cke.Operator, cke.OperationPhase) {
+func DecideOps(c *cke.Cluster, cs *cke.ClusterStatus, constraints *cke.Constraints, resources []cke.ResourceDefinition, rqEntries []*cke.RebootQueueEntry, newlyDrained []*cke.RebootQueueEntry, drainCompleted []*cke.RebootQueueEntry, drainTimedout []*cke.RebootQueueEntry, rebootDequeued []*cke.RebootQueueEntry) ([]cke.Operator, cke.OperationPhase) {
```

Comment:
I think there are too many arguments of DecideOps.

Can you put the arguments together into a structure?

Reply:
Which arguments are to be put together? All reboot queue entry slices (5 arguments) ?

Reply:
Arguments for reboot entry. (5 args)